library/include/ck/library/tensor_operation_instance/gpu/softmax/device_softmax_i8_i8_instance_type.hpp

```diff
@@ -34,6 +33,13 @@ using device_softmax_i8_i8_instances = std::tuple<
     // clang-format on
     >;
 
+template <index_t Rank, index_t Reduce>
+using device_softmax_i8_i8_generic_instance = std::tuple<
```

Comment:
I don't think we need to support softmax with int8 and F32 compute data type. 
Because this is not quantization version of softmax.
Non quantize softmax only output the value between 0 and 1.
Hence, the value is meaningless. 
Delete all Softmax with I8 F32 I8.

Reply:
Yes.  Removed by [4adc37](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/730/commits/e0e2207dc7c97aec0f24290bdc98b8dfa74adc37)